### PR TITLE
Enable leaf certificate refresh

### PR DESCRIPTION
**Pull request by swati-nair**

### CODE REVIEW

File: deploy/crds/operator.ibm.com_certmanagers_crd.yaml [hunk 1]
```diff
@@ -17,179 +17,197 @@ spec:
...
+                          resources required. If Requests is omitted for a container,
+                          it defaults to Limits if that is explicitly specified, otherwise
+                          to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/'
+                        type: object
+                    type: object
+                type: object
+              disableHostNetwork:
+                type: boolean
+              enableCertRefresh:
+                description: EnableCertRefresh is a flag that can be set to enable
```
Comment (tthorpe2) [inline deploy/crds/operator.ibm.com_certmanagers_crd.yaml:169]: Small suggested word change -              
enableCertRefresh:
                description: EnableCertRefresh is a flag that can be set to enable
                  **refresh of** leaf certificates based on a root CA

File: deploy/crds/operator.ibm.com_certmanagers_crd.yaml [hunk 2]
```diff
@@ -17,179 +17,197 @@ spec:
...
+              imagePostFix:
+                type: string
+              imageRegistry:
+                description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+                  Important: Run "operator-sdk generate k8s" to regenerate code after
+                  modifying this file Add custom validation using kubebuilder tags:
+                  https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html'
+                type: string
+              refreshCertsBasedOnCA:
+                description: RefreshCertsBasedOnCA is a list of CA certificate names
```
Comment (tthorpe2) [inline deploy/crds/operator.ibm.com_certmanagers_crd.yaml:183]: suggested word change here - Don't know if 2 sentences is considered good form but i think it reads better.

refreshCertsBasedOnCA:
                description: RefreshCertsBasedOnCA is a list of CA certificate names. Leaf certificates 
                  created from the CA will be refreshed when the CA is refreshed.

File: deploy/crds/operator.ibm.com_certmanagers_crd.yaml [hunk 3]
```diff
@@ -17,179 +17,197 @@ spec:
...
-                      description: 'Requests describes the minimum amount of compute
-                        resources required. If Requests is omitted for a container,
-                        it defaults to Limits if that is explicitly specified, otherwise
-                        to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/'
-                      type: object
+                    certName:
+                      type: string
+                    namespace:
+                      type: string
+                  required:
```
Comment (tthorpe2) [inline deploy/crds/operator.ibm.com_certmanagers_crd.yaml:191]: required means that if you enable this via EnableCertRefresh that these values are required?

File: deploy/crds/operator.ibm.com_certmanagers_crd.yaml (hunk 1 above)
Comment (swati-nair) [reply deploy/crds/operator.ibm.com_certmanagers_crd.yaml:169]: will update it. missed the word there.. thanks for catching it.

File: deploy/crds/operator.ibm.com_certmanagers_crd.yaml (hunk 3 above)
Comment (swati-nair) [reply deploy/crds/operator.ibm.com_certmanagers_crd.yaml:192]: Required means if `RefreshCertsBasedOnCA` field exists, then these two fields i.e. `certName` and `namespace` should exist.

**Review by tthorpe2 — APPROVED** [review]
/lgtm

**Pull request merged**